### PR TITLE
[5.2] scss comments

### PR DESCRIPTION
**Pull request by brianteeman**
Single line comments should begin with // followed by a space

This has zero effect on compiling the css. It is just for
- consistency
- easier to search for a comment without finding all the urls
- some scss tools require the space
- pedantic

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: build/media_source/templates/administrator/atum/scss/vendor/choicesjs/choices.scss [hunk 1]
```diff
@@ -43,7 +43,7 @@
   background-color: transparent;
 
   &::placeholder {
-    color: $choices-input-placeholder; //$gray-700; //var(--gray-200);
+    color: $choices-input-placeholder; //$gray-700; // var(--gray-200);
```
Comment (Quy) [inline build/media_source/templates/administrator/atum/scss/vendor/choicesjs/choices.scss]:
```suggestion
    color: $choices-input-placeholder; // $gray-700; // var(--gray-200);
```

File: build/media_source/templates/administrator/atum/scss/_variables.scss [hunk 2]
```diff
@@ -1,1033 +1,1035 @@
...
+// Remove prefix added in BS5 for compat with Joomla beta's shipped with Betas
+$prefix: "";
+
+// Use dark mode conditionally based on data-bs-theme attribute
+$color-mode-type: data;
+
+// Enable dark mode in J5.0
+$enable-dark-mode: true;
+
+// this is a test
```
Comment (wilsonge) [inline build/media_source/templates/administrator/atum/scss/_variables.scss]: Test noticed :)

**Comment by wilsonge** [comment]
LGTM and all whitespace changes now so merging on review.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**